pkgs/games/hmcl/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib, stdenv
+, fetchurl
+, jre
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hmcl";
+  version = "3.5.3.223";
+
+  src = ./.;
+
+  hmcl = fetchurl {
```

Comment:
```suggestion
  src = fetchurl {
```

Reply:
There's a problem with this, fetchurl doesn't know how to decompress the jar, and we don't want to unzip the jar